js/src/3rdparty/etherscan/util/address.js

```diff
@@ -0,0 +1 @@
+../../../abi/util/address.js
```

Comment:
? 

1. Why are we adding another file here?
2. There is no differences between having a reference to API vs. ABI. When using an exposed utility (unless you are the API), best to point to the API, not ABI.